dataflow/build.gradle

```diff
@@ -55,7 +55,12 @@ createDataflowShaded('nullaway')
 createDataflowShaded('errorprone')
 
 /**
- * Create a task to run a dataflow analysis demo
+ * Create a task to run a dataflow analysis testcase
+ * To add a dataflow analysis case, in normal cases you only
+ * need to add your test code directory and the classname and
+ * method {org.checkerframework.dataflow.util.DataflowTester.performTest}
+ * will receive your analysis instance. Go org.checkerframework.dataflow.util.DataflowTester
+ * to see more details if you want to make any change on it.
  * @param taskName the taskName for the created task
  * @param dirName the directory name of the test file
  * @param className the fully-qualified name of the test class
```

Comment:
It doesn't seem to be the fully-qualified name by looking at the live var example.

Reply:
Yeah, works fine.